samples/sensor/hx711/src/main.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2020 George Gkinis
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <sys/util.h>
+#include <sys/reboot.h>
+
+#include <device.h>
+#include <pm/device.h>
+#include <pm/device_runtime.h>
+#include <drivers/sensor.h>
+#include <drivers/sensor/hx711.h>
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(main, LOG_LEVEL_INF);
+
+const struct device *hx711;
+
+int reset(void)
+{
+	int ret = 0;
+
+	LOG_INF("Setting HX711_POWER_OFF");
+	ret = pm_device_runtime_put(hx711);
+	if (ret < 0) {
+		return ret;
+	}
+	k_msleep(1000);
+	LOG_INF("Setting HX711_POWER_ON");
+	ret = pm_device_runtime_get(hx711);
+	if (ret < 0) {
+		return ret;
+	}
+	return ret;
+}
```

Comment:
this should be done by the driver itself, see my previous comments.

Reply:
Removed and works.